pkg/client/client.go

```diff
@@ -281,4 +286,26 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, allAddresses,
 		}
 
 	}
+
+	if sr.Proxy != nil {
+		fmt.Fprintf(w, "Proxy Status:\tOK, ip %s, %d redirects, port-range %s\n",
+			sr.Proxy.IP, len(sr.Proxy.Redirects), sr.Proxy.PortRange)
+
+		if allRedirects {
+			for _, r := range sr.Proxy.Redirects {
+				fmt.Fprintf(w, "  Redirect %s, endpoint %d %s, %s %d->%d (created %s, last-updated %s)\n",
+					r.Protocol, r.EndpointID, r.EndpointLabels,
+					r.Location, r.Port, r.AllocatedProxyPort,
+					timeSince(time.Time(r.Created)),
+					timeSince(time.Time(r.LastUpdated)))
+				for _, rule := range r.Rules {
+					fmt.Fprintf(w, "    - %s\n", rule)
+				}
+				formatMessageForwardStatistics(w, "->", r.Statistics.Requests)
+				formatMessageForwardStatistics(w, "<-", r.Statistics.Responses)
+			}
+		}
+	} else {
+		fmt.Fprintf(w, "Proxy Status:\tNot running\n")
```

Comment:
If we have a `Status` in the API, this could print something more consistent with the other status fields, like for instance `Disabled`.

Reply:
I don't like `Disabled` because it is not a user decision to run the proxy or not. I've changed it to `No L7 rules loaded`